From 7c02e602a150d6cefd8ec9a9925a1fd42b948b52 Mon Sep 17 00:00:00 2001
From: dyep49 <dyep49@gmail.com>
Date: Fri, 15 Jun 2018 12:59:49 -0700
Subject: [PATCH] TRITON-492 Add metricPorts metadata to workflow for
 cmon-agent Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 boot/setup.sh | 5 ++++-
 package.json  | 2 +-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/boot/setup.sh b/boot/setup.sh
index 7f31f4a..8eede54 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -7,7 +7,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
@@ -52,6 +52,9 @@ sdc_log_rotation_add wf-runner /var/svc/log/*wf-runner*.log 1g
 sdc_log_rotation_add wf-backfill /var/svc/log/*wf-backfill*.log 1g
 sdc_log_rotation_setup_end
 
+# Add metadata for cmon-agent discovery
+mdata-put metricPorts 8881,8882
+
 # All done, run boilerplate end-of-setup
 sdc_setup_complete
 
diff --git a/package.json b/package.json
index 168e655..a63c0a2 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "workflow",
   "description": "SmartDataCenter Workflow API and Runners",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "repository": {
-- 
2.21.0

